common/lib/dependabot/shared_helpers.rb

```diff
@@ -13,6 +13,9 @@ module SharedHelpers
     BUMP_TMP_FILE_PREFIX = "dependabot_"
     BUMP_TMP_DIR_PATH = "tmp"
     GIT_CONFIG_GLOBAL_PATH = File.expand_path("~/.gitconfig")
+    USER_AGENT = "dependabot/#{Dependabot::VERSION} #{Excon::USER_AGENT} "\
+                 "ruby/#{RUBY_VERSION} (#{RUBY_PLATFORM}) "\
+                 "(+https://github.com/dependabot/dependabot-core)"
```

Comment:
Nit: this contact is fuzzy...

GitHub support would probably have better urgent response/escalation policies for managed Dependabot, but be _really_ confused if the problematic traffic is originating from someone using `dependabot-core` on a rando Jenkins.

So meh as this is the best idea, and any reference is better than "i am excon".

Reply:
Yeah also worried things might get lost/take a while through github support tbh